website/docs/getting-started/build-a-sample-app.mdx

```diff
@@ -46,16 +46,15 @@ To convert this simple command line application to a basic Yew web application,
 
 #### Update Cargo.toml
 
-Add `yew` to the list of dependencies.
+Add `yew` to the list of dependencies editing file:
```

Comment:
```suggestion
Add `yew` to the list of dependencies:
```

I think the original sentence is correct. The one in pull request is a run-on sentence.